cascadia.yaml

```diff
@@ -32,10 +32,16 @@ Options:
 
   - Name: Deli
     Type: string
-    Flag: d,delimiter
-    Usage: delimiter for pieces csv output
+    Flag: 'd,delimiter'
+    Usage: 'delimiter for pieces csv output'
     Value: "\\t"
 
+  - Name: NoHeader
+    Type: bool
+    Flag: 'no-header'
+    Usage: 'ignore headers when using --piece'
+    Value: false
```

Comment:
pls remove this line. `false` **is** the default value, so no need to spell it out. same goes for `--text` and `--raw-text`

Reply:
the `Value: false` line.